### DISCUSSION TITLE
Whether support Win7 by MiKTex

### DISCUSSION THREAD

**Discussion by lylgithub2021** [MiKTeX/miktex#839] [General]
The following description comes from MiKTex website:
quote:
MiKTeX for Windows requires one of the following Windows operating systems:

    Windows 10
    Windows Server 2016
    Windows 8/8.1 (all editions except RT)
    Windows Server 2012
end quote

Does that mean MiKTex will be not supporting `Windows 7` any longer?

**Comment by Tikatikatambo** [comment]
You wrote: "Does that mean MiKTex will be not supporting Windows 7 any longer?"

Are there earlier versions of MiKTex which do support Windows 7?

**Comment by Ulrich-Diez** [comment]
MiKTeX 2.9 supports Windows 7. But I don't know if there is a reliable non-compromised publicly available repository for MiKTeX 2.9 nowadays.

On the machine of an internet-friend of mine, still running under Windows 7, the MiKTeX 2.9 installation was destroyed when after years trying to update:

Instead of the update being aborted with a message about new MiKTeX binaries not supporting Windows 7, he ended up with a lot of binaries not running under Windows 7 but producing a bunch of errors.

(I myself bailed out some years ago and decided to switch from Windows to Linux/Debian and to use "vanilla" TeX Live but I still have a local repository of MiKTeX 2.9.5721 (64bit binaries), about 2GB, dating back to April 1, 2016, in my private archives, so after removal of the messy installation he could reinstall things. But updating binaries, using MiKTeX 2.9's update/package-manager, is not possible nowadays without turning the installation into something unusable. So there is no convenient way known to me for obtaining the last binaries that support Windows 7. Seems updating non-binary-packages/LaTeX-files (.sty-files etc) must be done by hand. And you don't have most recent LaTeX 2e-kernels/TeX-engines available unless you update formats and build executables "by hand".)

**Comment by Tikatikatambo** [comment]
Thanks so very much for this.

I am a fairly advanced PC user --- advanced enough to know that I don't
want MS nosing around my PC, but not advanced enough to know all the ins
and outs of the topic we are discussing.

Also, I have tried Linux and found it way worse than Windows. Actually,
Windows 7 works pretty well for me in most respects and I just want to
stick with it.

I am running an old version of SWP (3 point something) in XP Mode and
never upgraded. My old version serves me well, but I would like to
upgrade now that it is free. Do you think that SW 6 can be installed with
an older version of MiKTeX resulting in nothing more than some minor
typesetting limitations?

Thanks again, Sandy

On Thu, 28 Oct 2021 14:37:18 -0700 Ulrich-Diez ***@***.***>
writes:
> MiKTeX 2.9 supports Windows 7. But I don't know if there is a 
> reliable non-compromised publicly available repository for MiKTeX 
> 2.9 nowadays.
> 
> On the machine of an internet-friend of mine, still running under 
> Windows 7, the MiKTeX 2.9 installation was destroyed when after 
> years trying to update:
> 
> Instead of the update being aborted with a message about new MiKTeX 
> binaries not supporting Windows 7, he ended up with a lot of 
> binaries not running under Windows 7 but producing a bunch of 
> errors.
> 
> That's when I bailed out and decided to switch from Windows to Linux 
> and to use "vanilla" TeX Live.
> 
> (I still have a local repository of MiKTeX 2.9.5721 (64bit 
> binaries), about 2GB, dating back to April 1, 2016, in my private 
> archives, so after removal of the messy installation he could 
> reinstall things. But updating binaries, using MiKTeX 2.9's 
> update/package-manager, is not possible nowadays without turning the 
> installation into something unusable. So there is no convenient way 
> known to me for obtaining the last binaries that support Windows 7. 
> Seems updating non-binary-packages/LaTeX-files (.sty-files etc) must 
> be done by hand. And you don't have most recent LaTeX 
> 2e-kernels/TeX-engines available unless you update formats and build 
> executables "by hand".)
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
>
https://github.com/MiKTeX/miktex/discussions/839#discussioncomment-155562
2


____________________________________________________________
Choose to be safer online.
Opt-in to Cyber Safety with NortonLifeLock.
Get Norton 360 with LifeLock starting at $9.95/month.*
https://store.netzero.net/account/showService.do?serviceId=nz-nLifeLock&utm_source=mktg&utm_medium=taglines&utm_campaign=nzlifelk_launch&utm_content=tag995&promoCode=A23457

**Reply by Ulrich-Diez** [reply]
_> Also, I have tried Linux and found it way worse than Windows._ 

I *think* I know what you mean. :-)
(When I started with Linux, a big problem was the loads of contradicting information, and many bits of information not applying to the distribution/installation used by me... Besides this: Nowadays from "official" sides still often programs/apps are only available for Windows. Who wants to do it under Linux must see for themselves how she/he gets it done. E.g., device drivers, software for transferring images from digital camera to a volume attached to the computer, government software for submitting tax returns via the Internet,...)

For starting out and playing around and testing I recommend [KNOPPIX](http://www.knoppix.org/), a Live system which boots from USB-Stick/(micro-)SSD-card/DVD and thus does not affect Windows-installations present on the same machine's harddisk.... There is a nice LXDE-desktop and you also have things like Firefox- and Chromium-browser.

_> Actually, Windows 7 works pretty well for me in most respects and I just want to stick with it._

Windows 7 is my favorite Windows-operating-system in situations where I cannot avoid using Windows. (I did not yet see Windows 11. After all the unseriousness and harmful dictatorship over one's own computer and the disclosure of data that is thrown at you with Windows 10, I never want to have anything to do with Windows 10 again after seeing how it behaves on other people's machines.)

_> I am running an old version of SWP (3 point something) in XP Mode and never upgraded. My old version serves me well, but I would like to upgrade now that it is free._

I assume SWP means Scientific Workplace. :-)
I wasn't aware that Scientific Word 6.1 for Windows is free nowadays. Thank you for telling me :-)

_> Do you think that SW 6 can be installed with an older version of MiKTeX resulting in nothing more than some minor typesetting limitations?_

Sorry, I don't know. I never attempted to install/use Scientific Workplace/Scientific Word.

I just looked up Scientific Workplace and Scientific Word at [https://www.mackichan.com/index.html?products/swp.html](https://www.mackichan.com/index.html?products/swp.html) and learned that "Scientific **Word** 6.1 for Windows is now available for free". There are no statements about Scientific **Workplace**/Scientific **Notebook** in this regard.

On that page you find a link for downloading at [https://www.mackichan.com/techtalk/v60/1002updatesv6.htm](https://www.mackichan.com/techtalk/v60/1002updatesv6.htm).
That download-page also provides a link to a PDF-file "Getting Started, version 6" at [https://s3-us-west-2.amazonaws.com/mackichan-com-ftp/pub/download/version60/gettingstarted-60.pdf](https://s3-us-west-2.amazonaws.com/mackichan-com-ftp/pub/download/version60/gettingstarted-60.pdf).

There, in the section "Installation" I found the statement "This software requires Windows XP or above or Mac OS X 10.5 (Intel)."

All I can say is:

From this statement one can neither deduce nor rule out that attempting to install Scientific Word 6 on Windows 7, which probably in some aspects is "above" XP, and thus along with a MiKTeX-distribution like MiKTeX 2.9, which supports Windows 7, works out.

From this statement one can neither deduce nor rule out that attempting to install Scientific Word 6 on Windows 7, which probably in some aspects is "above" XP, and thus along with a MiKTeX-distribution like MiKTeX 2.9, which supports Windows 7, will damage a software- and operating-system-installation which previously was intact.

-----

If attempting this at all, I strongly recommend making a backup of your system before doing any changes so you can restore things in case something goes wrong/does not work out. 

For handling the case of needing to migrate my files/my data to another machine where a different operating-system/software-collection is installed I boot machines from external storage device with a Linux-Live-System (KNOPPIX) and use Linux's command-line tool [rsync](https://en.wikipedia.org/wiki/Rsync) for backing up files/directory structures to external harddrive.  From the rsync-backups things can easily be transferred/copied to the other machine. I think Windows' [robocopy](https://en.wikipedia.org/wiki/Robocopy) might suit that purpose as well.
When backing up data in a way suitable for handling the case of needing to migrate things to another machine, it is not always funny to get data out of programs that have their own internal data management. As I write this sentence, I am thinking of emails that are managed in the email program. And of data records for setting up e-mail accounts. Of electronic address books managed in the e-mail-program. And of web browsers with their own bookmark management. And much more.

For handling the case of needing to restore some state of operating-system- and software-installation on the same machine in case something goes wrong while doing an update or the like, I use [CloneZilla](https://clonezilla.org/), a live-system which boots from external storage device (CD/DVD/whatsoever external drive) and which is tailored to backing up and restoring/"mirroring" entire volumes/partitions, e.g., to external harddrive so that on the same machine I can easily restore some state of operating-system- and software-installation in case something went wrong.

For backing up and restoring I only use the already mentioned live-systems KNOPPIX (it brings along rsync) and CloneZilla, booting from CD/DVD. This way it is ensured that the operating-system in use for performing backup/restoration is not compromised and does not compromise backups. This way the software for backup/restoration is available in situations where a botched software installation caused the hard drive's operating-system to stop working. 

In situations where supply of electricity was rather volatile and damage due to overvoltage/electric shock was not entirely unlikely, as backup-device I used a RAID system with at least three backup-storage-devices for backups. I strictly ensured that never all three backup-storage-devices were physically connected to the RAID at the same time. When backing up, only backup-storage-device 1 was physically connected to the raid system. The system was backed up to backup-storage-device 1. Afterwards, the RAID was physically disconnected from the system to be backed up and  backup-storage-device 2 was physically connected to the RAID and synchronized with backup-storage-device 1. After that, backup-storage-device 2 was physically disconnected from the RAID and backup-storage-device 3 was physically connected to the RAID and synchronized with backup-storage-device 1. This way the likelihood of overvoltage/electric shock destroying both the system that was to be backed-up and all existing backups at the same time was decreased.

-----

Provided that

- you are interested in taking the risk of changing/damaging a running system/software-installation by attempting to install Scientific Word 6 and MiKTeX 2.9 on Windows 7 and
- you don't find another resource/repository where you can obtain MiKTeX 2.9 and
- you have 64-bit Windows 7 and
- you accept that I do not take any warranty or responsibility or liability or the like in any way whatsoever and
- you can make a suggestion on how to do the data-transfer (it is almost 2GB),

I can probably transfer a copy of my local offline-repository of MiKTeX 2.9 for 64bit-Windows-7 to you.
If transfer via internet is not feasible I could burn a DVD, put it in a paper-envelope and send it the traditional way...:-)

-----

**Edit on Friday, October 29, 2021, 21:07:55 UTC**

I just noticed TeX Live 2021 seems to support Windows 7.

[The TeX Live Guide—2021, section 1.2 Operating system support](1) says:

> TeX Live contains binaries for many Unix-based platforms, including GNU/Linux, Mac OS X, and Cygwin. The included sources can be compiled on platforms for which we do not provide binaries.
> 
> **As to Windows: Windows 7 and later are supported. Windows Vista may still mostly work, but TeX Live will no longer even install on Windows XP or earlier. TeX Live includes no 64-bit executables for Windows, but the 32-bit executables should run on 64-bit systems. But see [https://tug.org/texlive/windows.html](https://tug.org/texlive/windows.html) for options to add 64-bit binaries.**
> 
> See section 2.1 for alternate solutions for Windows and Mac OS X.

If Scientific Word 6.1  also works out with the most recent TeX Live 2021 instead of MiKTeX, this might probably be an option where you have a most recent TeX-distribution.

I found some installation instructions "Version 6.0 Fixed Licence Installation Instructions **with manual installation of TeXLive 2018** for Scientific Word / Scientific WorkPlace v6.0.28 and higher" at [https://www.sciword.co.uk/pdfs/v6.0ManualTeXLive.pdf](https://www.sciword.co.uk/pdfs/v6.0ManualTeXLive.pdf) and I found some installation instructions "Scientific Word/WorkPlace/Notebook Installation Instructions for version 6.x" at [https://www.sciword.co.uk/pdfs/v6.0Installation.pdf](https://www.sciword.co.uk/pdfs/v6.0Installation.pdf).

Maybe installing works out with TeXLive 2021 as well.

**Comment by Tikatikatambo** [comment]
Thanks for your response.

The system on which I put a dual boot system with Linux was completely
destroyed. Linux has an underlying incompatibility with Windows (I don't
remember the details) which made Windows inoperable and the only solution
was to completely wipe the HD clean so that Windows could be reinstalled.
I won't try Linux again in any way, shape or form. Also, Linux seems to
be intended for high tech and business applications and definitely not
for an individual computer user.

After decades of suffering through various incarnations of Windows. I
have found now that Windows 7 can be tamed so that it actually works
pretty well for me. There is a patching system called 0patch which can be
put on Windows 7 and which sends micropatches to fix vulnerabilities
identified in Windows more quickly and effectively than Windows Update. I
use this and common sense care in using Windows 7 (such as not opening
attachments or visiting  urls that come in spam email); so Windows 7
seems pretty safe to me.

Yes, SWP is Scientific Workplace. They cannot make SWP open source
because it contains mu-pad, a proprietary 3rd party program, but SW is
sufficient for my needs which are largely expository. If I need
calculations beyond what is available in SW, I can do them in my old
version of SWP and then open the result in the new program.

I am in no big hurry to get a newer version of SW operational, so I'll
just wait a while and see how things play out.

As a matter of safety, all my data files and work, including installable
applications are on a partition separate from my C drive, so in the very
worst catastrophic situation, none of that would be lost and only a fresh
install of the OS would be necessary.

Best wishes, Sandy Lemberg

On Fri, 29 Oct 2021 07:17:13 -0700 Ulrich-Diez ***@***.***>
writes:
> _> Also, I have tried Linux and found it way worse than Windows._ 
> 
> I *think* I know what you mean. :-)
> (When I started with Linux, a big problem was the loads of 
> contradicting information, and many bits of information not applying 
> to the distribution/installation used by me... Besides this: 
> Nowadays from "official" sides still often programs/apps are only 
> available for Windows. Who wants to do it under Linux must see for 
> themselves how she/he gets it done. E.g., device drivers, software 
> for transferring images from digital camera to a volume attached to 
> the computer, government software for submitting tax returns via the 
> Internet,...)
> 
> For starting out and playing around and testing I recommend 
> [KNOPPIX](http://www.knoppix.org/), a Live system which boots from 
> USB-Stick/(micro-)SSD-card/DVD and thus does not affect 
> Windows-installations present on the same machine's harddisk.... 
> There is a nice LXDE-desktop and you also have things like Firefox- 
> and Chromium-browser.
> 
> _> Actually, Windows 7 works pretty well for me in most respects and 
> I just want to stick with it._
> 
> Windows 7 is my favorite Window-OS in situations where I cannot 
> avoid using Windows. (I did not yet see Windows 11. After all the 
> unseriousness and harmful dictatorship over one's own computer and 
> the disclosure of data that is thrown at you with Windows 10, I 
> never want to have anything to do with Windows 10 again after seeing 
> how it behaves on other people's machines.)
> 
> _> I am running an old version of SWP (3 point something) in XP Mode 
> and never upgraded. My old version serves me well, but I would like 
> to upgrade now that it is free._
> 
> I assume SWP means Scientific Workplace. :-)
> I wasn't aware that Scientific Word 6.1 for Windows is free 
> nowadays. Thank you for telling me :-)
> 
> _> Do you think that SW 6 can be installed with an older version of 
> MiKTeX resulting in nothing more than some minor typesetting 
> limitations?_
> 
> Sorry, I don't know. I never attempted to install/use Scientific 
> Workplace/Scientific Word.
> 
> I just looked up Scientific Workplace and Scientific Word at 
>
[https://www.mackichan.com/index.html?products/swp.html](https://www.mack
ichan.com/index.html?products/swp.html) 
> and learned that "Scientific **Word** 6.1 for Windows is now 
> available for free". There are no statements about Scientific 
> **Workplace**/Scientific **Notebook** in this regard.
> 
> On that page you find a link for downloading at 
>
[https://www.mackichan.com/techtalk/v60/1002updatesv6.htm](https://www.ma
ckichan.com/techtalk/v60/1002updatesv6.htm).
> That download-page also provides a link to a PDF-file "Getting 
> Started, version 6" at 
>
[https://s3-us-west-2.amazonaws.com/mackichan-com-ftp/pub/download/versio
n60/gettingstarted-60.pdf](https://s3-us-west-2.amazonaws.com/mackichan-c
om-ftp/pub/download/version60/gettingstarted-60.pdf).
> 
> There, in the section "Installation" I found the statement "This 
> software requires Windows XP or above or Mac OS X 10.5 (Intel)."
> 
> All I can say is:
> 
> From this statement one can neither deduce nor rule out that 
> attempting to install Scientific Word 6 on Windows 7, which probably 
> in some aspects is "above" XP, and thus along with a 
> MiKTeX-distribution like MiKTeX 2.9, which supports Windows 7, works 
> out.
> 
> From this statement one can neither deduce nor rule out that 
> attempting to install Scientific Word 6 on Windows 7, which probably 
> in some aspects is "above" XP, and thus along with a 
> MiKTeX-distribution like MiKTeX 2.9, which supports Windows 7, will 
> damage a software- and OS-installation which previously was intact.
> 
> -----
> 
> If attempting this at all, I strongly recommend making a backup of 
> your system before attempting this so you can restore things in case 
> something goes wrong/does not work out. 
> 
> For handling the case of needing to migrate my files/my data to 
> another machine where a different OS/software-collection is 
> installed I boot machines from external storage device with a 
> Linux-Live-System (KNOPPIX) and use Linux's command-line tool 
> [rsync](https://en.wikipedia.org/wiki/Rsync) for backing up files to 
> external harddrive.  From the rsync-backups things can easily be 
> transferred/copied to the other machine. I think Windows' 
> [robocopy](https://en.wikipedia.org/wiki/Robocopy) might suit that 
> purpose as well.
> When backing up data in a way suitable for handling the case of 
> needing to migrate things to another machine, it is not always funny 
> to get data out of programs that have their own internal data 
> management. As I write this sentence, I am thinking of emails that 
> are managed in the email program. And of data records for setting up 
> e-mail accounts. Of electronic address books managed in the 
> e-mail-program. And of web browsers with their own bookmark 
> management. And much more.
> 
> For handling the case of needing to restore some state of OS- and 
> software-installation on the same machine in case something goes 
> wrong while doing an update or the like, I use 
> [CloneZilla](https://clonezilla.org/), a Live-System which boots 
> from external storage device (CD/DVD/whatsoever external drive) and 
> which is tailored to backing up and restoring/"mirroring" entire 
> volumes/partitions, e.g., to external harddrive so that on the same 
> machine I can easily restore some state of OS- and 
> software-installation in case something went wrong.
> 
> For backing up and restoring I only use the already mentioned 
> Live-Systems KNOPPIX (it brings along rsync) and CloneZilla, booting 
> from CD/DVD. This way it is ensured that the operating system in use 
> for performing backup/restoration is not compromised and does not 
> compromise backups. This way the software for backup/restoration is 
> available in situations where a botched software installation caused 
> the hard drive's operating system to stop working. 
> 
> In situations where supply of electricity was rather volatile and 
> damage due to overvoltage/electric shock was not entirely unlikely, 
> as backup-device I used a RAID system with at least three 
> backup-storage-devices for backups. I strictly ensured that never 
> all three backup-storage-devices were physically connected to the 
> RAID at the same time. When backing up, only backup-storage-device 1 
> was physically connected to the raid system. The system was backed 
> up to backup-storage-device 1. Afterwards, the RAID was physically 
> disconnected from the system to be backed up and  
> backup-storage-device 2 was physically connected to the RAID and 
> synchronized with backup-storage-device 1. After that, 
> backup-storage-device 2 was physically disconnected from the RAID 
> and backup-storage-device 3 was physically connected to the RAID and 
> synchronized with backup-storage-device 1. This way the likelihood 
> of overvoltage/electric shock destroying both the system that was to 
> be backed-up and all existing backups at the same time was 
> decreased.
> 
> -----
> 
> Provided that
> - you are interested in taking the risk of changing/damaging a 
> running system/software-installation by attempting to install 
> Scientific Word 6 and MiKTeX 2.9 on Windows 7
> - you don't find another resource/repository where you can obtain 
> MiKTeX 2.9 and
> - you have 64-bit Windows 7 and
> - you accept that I do not take any warranty or responsibility or 
> liability or the like in any way whatsoever 
> - you can make a suggestion on how to do the data-transfer (it is 
> almost 2GB)
> I can probably transfer a copy of my local offline-repository of 
> MiKTeX 2.9 for 64bit-Windows-7 to you.
> (If transfer via internet is not feasible I could burn a DVD, put it 
> in a paper-envelope and send it the traditional way...)
> 
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
>
https://github.com/MiKTeX/miktex/discussions/839#discussioncomment-155926
3=


____________________________________________________________
Choose to be safer online.
Opt-in to Cyber Safety with NortonLifeLock.
Plans starting as low as $6.95 per month.*
https://store.netzero.net/account/showService.do?serviceId=nz-nLifeLock&utm_source=mktg&utm_medium=taglines&utm_campaign=nzlifelk_launch&utm_content=tag695&promoCode=A34454

**Reply by Ulrich-Diez** [reply]
_> I am in no big hurry to get a newer version of SW operational, so I'll just wait a while and see how things play out._

In case you are interested in obtaining my MiKTeX 2.9-offline-repository for 64bit-Windows 7 let me know.
[ In case transfer via internet is not an option, I think once a year I can afford the luxury of letter postage for an international letter. :-)) ]
(But I think SW6 should cooperate with TeX Live 2021 - which claims to support Windows 7 - as well.)

_> The system on which I put a dual boot system with Linux was completely destroyed. Linux has an underlying incompatibility with Windows_ 

I'm sorry you had a bad experience.

In primeval IT-times - 90s of the last century/generation 28.8K - I had Windows95/98 parallel to SuSE-Linux. Windows was installed on active primary partition on the hard-disk at the IDE-controller. Linux was installed on active primary partition on the hard-disk at the SCSI-controller. The operating-systems didn't really interfere. Unlike my friends I didn't even need a boot-manager like LILO at all. Just pressed F12 when during the boot-up the BIOS told me to do so for selecting another boot-device than the default and then a menu opened up where one could chose the hard-drive where to boot from...  Life was simple back then. :->

Nowadays where we have no more BIOS-machines with MBR partitions but UEFI and GPT it easily happens that a layman is lost in the automatized chaos and cannot prevent damage, e.g., when installation of the Grub2-bootmanager or the like is about to go wrong... The last time I was involved in dual-boot was in 2014 with Windows 7 and Ubuntu 15.04. For Windows 7 in the BIOS the quick-boot-feature and the secure-boot-feature needed to be disabled... All in all it took us about 7 hours of fiddling around until things worked out. :->

_> As a matter of safety, [...]_

Then I don't have to worry. :-) (Sometimes I get a little carried away when people are considering changing the software installation on their computer, hereby entering "uncharted territory". Then I write a lot, because I often witness friends and colleagues trying things without proper backups...)

I wish you all the best and no computer problems. :-)

Sincerely

Ulrich

**Comment by Tikatikatambo** [comment]
If I remember correctly, the presence of Linux forced Windows to display
Linux time which is incompatible with Linux time. There was likely more
to it than that. In any case, I would like to stick with Windows 7 for
the reasons I described earlier.

Incidentally, if you are interested, I can provide you with the details
of how to tame Windows 7 to make it user friendly.

I am interested in your MiKTeX 2.9-offline-repository for 64bit-Windows 7
(although I don't know what you mean by "repository"). If I can get SW6
to work, that would be great.

Can this be done with Dropbox? One trick I have learned is that .exe
files can be shared by changing the .exe extension to .txt (the recipient
then changes back to .exe).

Are you in Germany? I am in the US. It would be good to exchange email
addresses if that can be done privately.

Thanks again, Sandy Lemberg

On Sun, 31 Oct 2021 10:22:55 -0700 Ulrich-Diez ***@***.***>
writes:
> _> I am in no big hurry to get a newer version of SW operational, so 
> I'll just wait a while and see how things play out._
> 
> In case you are interested in obtaining my MiKTeX 
> 2.9-offline-repository for 64bit-Windows 7 let me know.
> [ In case transfer via internet is not an option, I think once a 
> year I can afford the luxury of letter postage for an international 
> letter. :-)) ]
> (But I think SW6 should cooperate with TeX Live 2021 - which claims 
> to support Windows 7 - as well.)
> 
> _> The system on which I put a dual boot system with Linux was 
> completely destroyed. Linux has an underlying incompatibility with 
> Windows_ 
> 
> I'm sorry you had a bad experience.
> 
> In primeval IT-times - 90s of the last century/generation 28.8K - I 
> had Windows95/98 parallel to SuSE-Linux. Windows was installed on 
> active primary partition on the hard-disk at the IDE-controller. 
> Linux was installed on active primary partition on the hard-disk at 
> the SCSI-controller. The operating-systems didn't really interfere. 
> Unlike my friends I didn't even need a boot-manager like LILO at 
> all. Just pressed F12 when during the boot-up the BIOS told me to do 
> so for selecting another boot-device than the default and then a 
> menu opened up where one could chose the hard-drive where to boot 
> from...  Life was simple back then. :->
> 
> Nowadays where we have no more BIOS-machines with MBR partitions but 
> UEFI and GPT it easily happens that a layman is lost in the 
> automatized chaos and cannot prevent damage, e.g., when installation 
> of the Grub2-bootmanager or the like is about to go wrong... The 
> last time I was involved in dual-boot was in 2014 with Windows 7 and 
> Ubuntu 15.04. For Windows 7 in the BIOS the quick-boot-feature and 
> the secure-boot-feature needed to be disabled... All in all it took 
> us about 7 hours of fiddling around until things worked out. :->
> 
> _> As a matter of safety, [...]_
> 
> Then I don't have to worry. :-) (Sometimes I get a little carried 
> away when people are considering changing the software installation 
> on their computer, hereby entering "uncharted territory". Then I 
> write a lot, because I often witness friends and colleagues trying 
> things without proper backups...)
> 
> I wish you all the best and no computer problems. :-)
> 
> Sincerely
> 
> Ulrich
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
>
https://github.com/MiKTeX/miktex/discussions/839#discussioncomment-156645
2

____________________________________________________________
Sponsored by https://www.newser.com/?utm_source=part&utm_medium=uol&utm_campaign=rss_taglines_more

Brooke Shields Addresses Infamous Calvin Klein Ad
http://thirdpartyoffers.juno.com/TGL3131/617ed9f52e29759f419f5st01vuc1
Over 1K Flights Canceled Due to Weather, Staffing
http://thirdpartyoffers.juno.com/TGL3131/617ed9f54ce1c59f419f5st01vuc2
Billionaire Breaks California Real Estate Record
http://thirdpartyoffers.juno.com/TGL3131/617ed9f56cdd759f419f5st01vuc3

**Reply by Ulrich-Diez** [reply]
_> I am interested in your MiKTeX 2.9-offline-repository for 64bit-Windows 7_
_> (although I don't know what you mean by "repository")._ 

When in April 2016 I downloaded and executed the installer for MiKTeX 2.9 for 64-bit Windows, the installer offered the possibility of storing on local hard-disk every file that was about to be downloaded - within a so-called "local repository" so that you could install MiKTeX on more than one machine without having to download from repository-servers in the Internet again and again.
That "local repository" is just a folder where all downloaded files are stored.
The term "local repository" (in contrast to the remote-repository-servers that were to be accessed via internet) is used in the dialogue-boxes of the installer for MiKTeX 2.9 for 64-bit Windows.
The installer lets you choose whether you wish to download from a server in the internet (which is not possible any more nowadays) or whether to use a so-called local repository.
When choosing "local repository" a file-browser-window opens up and you are asked to browse to the folder on your local computer were all the downloaded files are stored.

_> If I can get SW6 to work, that would be great._

Probably you can get SW6 to work with the Windows-release of TeX Live 2021 instead of MiKTeX as well.
The advantage would be a TeX-installation which is up to date (LaTeX 2e-kernels with expl3/xparse integrated, most recent TikZ/pdf etc) and which - unlike MiKTeX 2.9 - can still be updated.

_> Can this be done with Dropbox?
One trick I have learned is that .exe
files can be shared by changing the .exe extension to .txt (the recipient
then changes back to .exe)._

I never used Dropbox until now.
**So I just registered a free Dropbox-account and uploaded everything there.**

The only .exe-file is the installer, setup-2.9.5721-x64.exe. I copied it to setup-2.9.5721-x64_exe.txt.
Everything else is .tar.lzma or .tar.bz2 or .cab or .ini or .txt.

**So I think, yes, things can be done via Dropbox. :-)**

_> Are you in Germany?_

Yes.

_>  It would be good to exchange email addresses if that can be done privately._

I have an e-mail-address which I use as from-header when posting to usenet-newsgroup comp.text.tex / de.comp.text.tex, i.e., an e-mail-address which is "fodder for the public" and which gets a lot of spam as well:
You can reach me at   eu_angelion[at]web.de  .
(When I set it up, this address was meant as a joke: Since "eu angelion" literally means "good news", I couldn't resist.)

Sincerely

Ulrich

**Reply by Ulrich-Diez** [reply]
> If I remember correctly, the presence of Linux forced Windows to display Linux time which is incompatible with Linux time.

I just stumbled over these:

[How to Fix Windows and Linux Showing Different Times When Dual Booting](https://www.howtogeek.com/323390/how-to-fix-windows-and-linux-showing-different-times-when-dual-booting/)

[Correct time for dual boot with Windows](https://www.tuxedocomputers.com/en/Infos/Help-Support/Instructions/Correct-time-for-dual-boot-with-Windows.tuxedo#)

Sincerely

Ulrich

**Comment by Tikatikatambo** [comment]
Dear Ulrich

I have been very busy, so I apologize for the delay in responding to you.

I am interested in what you have put on Dropbox. However, for me to
access it, you need to grant me access. The instructions for this are on
the Dropbox site.

Please use my email address to grant me access. Let me know if you don't
have it, and I'll provide it using an email address you have sent me.

I won't have time to try it for a while, but I would like to get the
files on my PC so that I can access them when I need to.

Regarding Linux, I really have no interest in this for the reasons I gave
you earlier. I am satisfied with Windows 7 (other than the issue with
MiKTeX which we have been discussing). Other than this MiKTeX issue,
Windows 7 does not bother me at all, since I have found ways to tame it.

As a point of information, I went to the local LUG and found that they
were concerned exclusively with matters in which I have ZERO interest
(mainly Android).

Thanks again, Sandy

On Tue, 02 Nov 2021 17:02:22 -0700 Ulrich-Diez ***@***.***>
writes:
> > If I remember correctly, the presence of Linux forced Windows to 
> display Linux time which is incompatible with Linux time.
> 
> I just stumbled over this:
> 
> [Correct time for dual boot with 
>
Windows](https://www.tuxedocomputers.com/en/Infos/Help-Support/Instructio
ns/Correct-time-for-dual-boot-with-Windows.tuxedo#)
> 
> Sincerely
> 
> Ulrich
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
>
https://github.com/MiKTeX/miktex/discussions/839#discussioncomment-157936
2


____________________________________________________________
Choose to be safer online.
Opt-in to Cyber Safety with NortonLifeLock.
Get Norton 360 with LifeLock starting at $9.95/month.*
https://store.netzero.net/account/showService.do?serviceId=nz-nLifeLock&utm_source=mktg&utm_medium=taglines&utm_campaign=nzlifelk_launch&utm_content=tag995&promoCode=A23457

**Reply by Ulrich-Diez** [reply]
Dear Sandy!

> Dear Ulrich I have been very busy, so I apologize for the delay in responding to you.

Never mind and definitely no need to apologize. :-) Life keeps us busy. As you said yourself, this is not an urgent matter. No need to put ourselves under time pressure. :-) 

> However, for me to access it, you need to grant me access.

I know. :-)

> Please use my email address to grant me access. Let me know if you don't have it, and I'll provide it using an email address you have sent me

I don't have your e-mail address. :-)
(All I have is:  notifications[at]github.com  )
If I had your e-mail-address I would have granted access to you already in the course of uploading things to my Dropbox.

I suggest the following proceeding:

Please let me know your e-mail-address by contacting me via my e-mail-address eu_angelion[at]web.de .
I suggest in your e-mail you use a subject-line containing the phrase "MiKTeX".
In the text of your e-mail please also suggest a subject-line which I shall use in my reply.

As soon as I have knowledge of your e-mail-address I will grant access to you/to your e-mail-address and send a Dropbox-download-link to your e-mail-address, using exactly the subject-line suggested by you, i.e., not preceded by an additional "RE:" or the like.

As soon as you have successfully downloaded everything, please let me know via eu_angelion[at]web.de so I can remove the files from Dropbox and close the dropbox-account.

In case of problems, please let me know via eu_angelion[at]web.de so that we can coordinate the further proceedings in this case.

(I prefer sending e-mail in text-format, no html-content and the like.)

Sincerely

Ulrich
